s3/README.md

```diff
@@ -0,0 +1,41 @@
+Quick Start
```

Comment:
sorry, but I don't see a header at the top of this file still. maybe a local commit that hasn't been pushed yet?